mapbox/libjava-core/src/main/java/com/mapbox/services/Constants.java

```diff
@@ -26,6 +26,7 @@
   public static final int GOOGLE_PRECISION = 5;
   public static final int OSRM_PRECISION_V4 = 6;
   public static final int OSRM_PRECISION_V5 = 5;
+  public static final int OSRM_PRECISION_6_V5 = 6;
```

Comment:
@cammace Let's not do this, let's use `PRECISION_6 ` from https://github.com/mapbox/mapbox-java/pull/312 instead.